anime-list-master.xml

```diff
@@ -53,6 +53,12 @@
   </anime>
   <anime anidbid="12" tvdbid="72070" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Chobits</name>
+	<mapping-list>
+	  <mapping anidbseason="0" tvdbseason="0">;1-3;2-4;</mapping>
+	  <mapping anidbseason="1" tvdbseason="1" start="10" end="17" offset="-1"/>
+	  <mapping anidbseason="1" tvdbseason="1" start="18" end="26" offset="-2"/>
```

Comment:
Wouldn't 19 to 26 be the correct values?

Reply:
Yea that makes sense, we don't need to offset the inserted recap episode. Just tested it out and that looks good, didn't have to mess around to get the thumbnails to line up right like before. 

Fixed and pushed both changes up